Automattic/happychat-client
# Custom Fields: Send form fields and channel

Sets up code to send Custom Field data about the chat into Happy Chat. For now this just sends in the `channel` (product) the chat is being assigned to, but it can be used for other data in the future.

More about Custom Fields: pbvpgB-UE-p2

### How to test
- Run the `happychat` service locally from the `try/chat-data` branch
- Run the HUD locally from the `try/chat-data` branch
- Run `happychat-client` locally from the `try/chat-data` branch
- When you open a chat from the client, you should see some Custom Fields in the HUD's right sidebar
